app/src/org/odk/collect/android/jr/extensions/PollSensorAction.java

```diff
@@ -100,6 +101,8 @@ public void run() {
                 requestLocationUpdates(providers);
             }
         });
+
+        return null;
```

Comment:
This returns null because the actual value setting happens in a callback. This isn't ideal, but shouldn't interfere with how the result of `processAction` is used in `FormDef`